image-assets/azure.sh

```diff
@@ -1,17 +1,24 @@
 #!/bin/bash
 
+set -ex
 # Transform a raw image disk to azure vhd
```

Comment:
If the new go implementation works, shouldn't we delete this script altogether?

Reply:
no this is a different script, this doesnt cover the transformation yet to gce/vhd